docs/util/schema.go

```diff
@@ -0,0 +1,642 @@
+package util
```

Comment:
naming a package just util is not best practice. If you look at the io package it actually uses io.ioutils. This way even if the package is standalone it will be clear, what it does.

Reply:
Could name it `schema`, for example. And name the struct inside `Data`, this way you can call it `schema.Data` from other packages instead of `schema.SchemaData` or `util.SchemaData`

Reply:
@stp-ip @alarsyo Thanks, noted. Named util considering the schema.go file as the utility file the generator but I agree now.